src/components/subPages/taskData.scss

```diff
@@ -0,0 +1,51 @@
+section {

```

Comment:
Dodając taki selector ostylowujesz wszystkie tagi  `section` na stronie. To może powodować problemy gdy użyjesz ponownie tagu section a będziesz chciał inaczej go ostylować bo będzie trzeba resetować style stąd.